libnavui-androidauto/src/main/java/com/mapbox/androidauto/navigation/lanes/CarLanesImageRenderer.kt

```diff
@@ -7,7 +7,6 @@ import androidx.car.app.model.CarIcon
 import androidx.car.app.navigation.model.Step
 import com.mapbox.navigation.ui.maneuver.api.MapboxLaneIconsApi
 import com.mapbox.navigation.ui.maneuver.api.MapboxManeuverApi
-import com.mapbox.navigation.utils.internal.ifNonNull
```

Comment:
this is unrelated but i searched for other `.internal` api imports and removed them.

I'd like to add a linter for this but haven't gotten around to it